library/pom.xml

```diff
@@ -18,7 +18,7 @@
 		<bootstrap-version>3.3.7</bootstrap-version>
 		<font-awesome-version>4.7.0</font-awesome-version>
 		<!-- Default extra plugins enabled -->
-		<ckeditor-extra-plugins>sakaipreview,image2,audiorecorder,contentitem,html5video,wordcount,notification,autosave,bt_table</ckeditor-extra-plugins>
+		<ckeditor-extra-plugins>sakaipreview,</ckeditor-extra-plugins>
```

Comment:
I believe some of these were here because this is a property that could be configured as part of the build. I'm not sure if anyone was doing this to change the plugins. It **looks** like this change makes these default so they can't be disabled anymore, any maybe that's fine.

Reply:
great point